conf/test_targeted.config

```diff
@@ -25,4 +25,7 @@ params {
 
     // Aligner
     aligner = 'minimap2'
+
+    // Steps
+    overrepresented = true
```

Comment:
```suggestion
    over_represented = true
```

Reply:
⚠️ the parameter is now called `overrepresented`, if we change it here we have to change it in `nextflow.config` and wherever `params.overrepresented` is used